server/routes/contacts/manage/contact-search/contactSearchController.ts

```diff
@@ -20,7 +20,7 @@ export default class ContactSearchController implements PageHandler {
     const journey = req.session.addContactJourneys[journeyId]
     const validationErrors = res.locals.validationErrors?.search
     const page = Number(req.query.page as unknown) || 0
-    const pageSize = config.apis.prisonerSearchApi.pageSize || 20
+    const pageSize = config.apis.contactsApi.pageSize || 10
     let results = null
```

Comment:
Maybe the page-size defaults would be better set in the config.ts file itself rather than in the controller code?